templates/kubernetes/terraform/modules/kubernetes/backend_service.tf

```diff
@@ -0,0 +1,24 @@
+data "aws_secretsmanager_secret" "by-name" {
```

Comment:
The name of this should be something like `cf_keypair` instead of `by-name`

Reply:
Are you sure?
I looked at the docs closely and it doesn't seem to be that way:

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/secretsmanager_secret
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/secretsmanager_secret_version

I am completely new to this, however!

Reply:
Ah, I guess they're using a made-up-name that also explains the purpose of the example they're implementing.
Sorry about that.
Easy to misread...

Reply:
Yeah. The syntax of `data` and `resource` in terraform always looks like `data "<type of thing you are referencing>" "<the name you are giving the thing so you can refer to it later>"`

Reply:
Makes sense it would be consistent. :-)